<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientLoginGUI</class>
 <widget class="QWidget" name="clientLoginGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="loginBox">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>100</y>
     <width>200</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>LOGIN</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>USERNAME:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loginUsername">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>PASSWORD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="loginPassword">
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 0, 0)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Hover" name="loginButton">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>28</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>220</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>430</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>HELP</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Hover</class>
   <extends>QPushButton</extends>
   <header location="global">hover.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
